Update things-everyone-must-know.md (#554)
mattermost/mattermost-handbook: contributors/onboarding/things-everyone-must-know.md

> 1. **Always use the term "country/region" and never "country"** in discussions, documentation and software design. Mattermost, Inc. may work with public sector organizations with a high degree of sensitivity to regions where the label "country" may be controversial.  

Reviewer: ```suggestion
1. **Always use the term "country/region" and never "country"** in discussions, documentation, and software design. Mattermost, Inc. may work with public sector organizations with a high degree of sensitivity to regions where the label "country" may be controversial.
```

> 2. **Never sign on behalf of Mattermost, Inc.** Only the Mattermost, Inc. CEO can sign documents on behalf of the company, with the one and only exception being signing an on-site NDA \(non-disclosure agreement\) in order to enter a building. See [Risk Management in Agreements](../../operations/finance/risk-management/) documentation for instructions on having any agreement properly reviewed and initialed by someone Director-level or higher prior to CEO signature. 

Reviewer: ```suggestion
2. **Never sign on behalf of Mattermost, Inc.** Only the Mattermost, Inc. CEO can sign documents on behalf of the company, with the one and only exception being signing an on-site NDA \(non-disclosure agreement\) in order to enter a building. See [Risk Management in Agreements](../../operations/finance/risk-management/) documentation for instructions on having any agreement properly reviewed and initialed by someone Director-level or higher prior to CEO signature.
```

3. **Escalate sensitive topics** seen in public sites such as the forums to the [Community Escalations channel](https://community.mattermost.com/private-core/channels/community-escalations), and follow the [Community Response Policy](../../operations/operations/company-policies/community-response-policy).

> 4. **Outside work** As an open-source company, Mattermost encourages Contributors to work on projects that are unrelated to Mattermost's business. Contributors do not require prior approval from Mattermost to work on projects that are unrelated to Mattermost's business or that do not pose a conflict of interest. Contributors should reference Mattermost's PIAA or contact hr@mattermost.com with questions. 

Reviewer 2: ```suggestion
4. **Outside work:** As an open-source company, Mattermost encourages Contributors to work on projects that are unrelated to Mattermost's business. Contributors do not require prior approval from Mattermost to work on projects that are unrelated to Mattermost's business or that do not pose a conflict of interest. Contributors should reference Mattermost's PIAA or contact hr@mattermost.com with questions.
```

> 4. **Outside work:** As an open-source company, Mattermost encourages Contributors to work on projects that are unrelated to Mattermost's business. Contributors do not require prior approval from Mattermost to work on projects that are unrelated to Mattermost's business or that do not pose a conflict of interest. Contributors should reference Mattermost's PIAA or contact hr@mattermost.com with questions.

Reviewer: ```suggestion
4. **Outside work:** As an open source company, Mattermost encourages Contributors to work on projects that are unrelated to Mattermost's business. Contributors do not require prior approval from Mattermost to work on projects that are unrelated to Mattermost's business or that do not pose a conflict of interest. Contributors should reference Mattermost's PIAA or contact hr@mattermost.com with questions.
```

> FT Staff may work on outside projects (paid or unpaid) that are not related to Mattermost's business, do not represent a conflict of interest, and do not hinder their ability to fulfill their contractual obligations to Mattermost. Prior to commencing work on outside projects, staff must confirm no conflict of interest exists by obtaining written approval from their manager and MLT. 

Reviewer 2: Unsure if we want to write out "FT" and "MLT" instead of use abbreviations.

Reviewer: Updated "FT" but we use MLT as-is in the docs so 3/5 on it remaining as such. What do you think?

> FT Staff may work on outside projects (paid or unpaid) that are not related to Mattermost's business, do not represent a conflict of interest, and do not hinder their ability to fulfill their contractual obligations to Mattermost. Prior to commencing work on outside projects, staff must confirm no conflict of interest exists by obtaining written approval from their manager and MLT. 

Reviewer: ```suggestion
Full-time staff may work on outside projects (paid or unpaid) that are not related to Mattermost's business, do not represent a conflict of interest, and do not hinder their ability to fulfill their contractual obligations to Mattermost. Prior to commencing work on outside projects, staff must confirm no conflict of interest exists by obtaining written approval from their manager and MLT.
```

> To obtain approval, email your manager and copy hr@mattermost.com to request approval. Include the following: details of the project/work that you intend to work on; length of obligation (if known); approximate weekly or monthly time committed to working on outside project (if known).

Reviewer 2: ```suggestion
To obtain approval, email your manager and copy hr@mattermost.com to request approval. Include the following: details of the project/work that you intend to work on; length of obligation (if known); and an approximate weekly or monthly time committed to working on outside project (if known).
```

> To obtain approval, email your manager and copy hr@mattermost.com to request approval. Include the following: details of the project/work that you intend to work on; length of obligation (if known); and an approximate weekly or monthly time committed to working on outside project (if known).

Reviewer: ```suggestion
To obtain approval, email your manager and copy hr@mattermost.com to request approval. Include the following: details of the project/work that you intend to work on; length of obligation (if known); and the approximate weekly or monthly time commitment to working on the outside project (if known).
```

> Manager and MLT approval will be stored in the staff's Bamboo documents file by HR. 

Reviewer: ```suggestion
Manager and MLT approval will be stored by HR in the staff member's BambooHR documents file.
```
